src/xbps/packages/management.md

```diff
@@ -4,11 +4,15 @@ Most general package management is done with the following commands:
 
 - [xbps-query(1)](https://man.voidlinux.org/xbps-query.1) displays information
    about packages installed locally, or, with the `-R` flag, packages contained
-   in repositories.
+   in repositories. It can also be used to search for packages.
```

Comment:
I don't feel strongly but maybe we could say that `xbps-query` searches for and displays information...?

Reply:
Feels more concise. I think it would be better.

Reply:
I made another small change in this paragraph, to make the part about the `-R` flag clearer.